proxy/http/HttpConfig.cc

```diff
@@ -1129,6 +1129,8 @@ HttpConfig::startup()
   // Local Manager
   HttpEstablishStaticConfigLongLong(c.synthetic_port, "proxy.config.admin.synthetic_port");
 
+  // FTP protocol enabled
+  HttpEstablishStaticConfigByte(c.forward_proxy_ftp_enabled, "proxy.config.ftp_enabled");
```

Comment:
Can we make this overridable as well? That would allow you to turn on /off this feature per remap rule (or via a plugin).